Cubical/Categories/Instances/PosetCat/Monotone.agda

```diff
@@ -0,0 +1,207 @@
+{-# OPTIONS --safe #-}
+
+-- TODO: This could be generalized to handle monotone functions on
+-- both preorders and posets.
```

Comment:
Yeah I think my preference would be to define Preorder as the basic category and define Poset as a `Constructions.FullSubcategory`

Reply:
Opened #32 to track this refactor.